src/utils/viewer/viewer.cpp

```diff
@@ -633,66 +676,27 @@ Viewer::Viewer(const Arguments& arguments)
   Mn::GL::Renderer::enable(Mn::GL::Renderer::Feature::DepthTest);
   Mn::GL::Renderer::enable(Mn::GL::Renderer::Feature::FaceCulling);
 
-  sceneFileName = args.value("scene");
-  bool useBullet = args.isSet("enable-physics");
-  if (useBullet && (args.isSet("debug-bullet"))) {
+  // whether to cycle scene instances
+  cycleSceneInstances_ = args.isSet("cycle-scenes");
```

Comment:
Can't we cycle on any scene dataset even if we specify to start with a particular one? Maybe the cycle-scenes replaces the default `-- scene` as a mutually exclusive option to override the initial, but otherwise behaves the same once loaded?

Reply:
That would require reloading a different scene-dataset and all the constituent metadata.  May as well restart.

Reply:
Oops, misunderstood.  This is a good idea - make it always available.